docs/transformers/multibyte-text-normalizer.md

```diff
@@ -0,0 +1,23 @@
+<span style="float:right;"><a href="https://github.com/RubixML/RubixML/blob/master/src/Transformers/MultibyteTextNormalizer.php">[source]</a></span>
+
+# Multibyte Text Normalizer
+This transformer converts all multibyte text to lowercase and removes extra whitespace.
+
+**Interfaces:** [Transformer](api.md#transformer)
+
+**Data Type Compatibility:** Categorical
+
+⚠ This normalizer is slower than the non multibyte [TextNormalizer](text-normalizer.md). Using the multibyte string polyfill will also strongly reduce performance of the transformer. We recommend to install the [multibyte string extension](https://www.php.net/manual/en/book.mbstring.php).
```

Comment:
I think we can simplify this note to just "We recommend you to install the Multibyte string extension for best performance."